src/modules/dashboard/components/ActionsPage/ActionsPageFeedItem/ActionsPageFeedItem.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import TransactionMeta from '../TransactionMeta';
+import { Address } from '~types/index';
+import TextDecorator from '~lib/TextDecorator';
+import UserMention from '~core/UserMention';
+import HookedUserAvatar from '~users/HookedUserAvatar';
+import { getMainClasses } from '~utils/css';
+
+import styles from './ActionsPageFeedItem.css';
+
+const displayName = 'dashboard.ActionsPage.ActionsPageFeedItem';
+
+interface Props {
+  comment: string;
+  username: string;
+  walletAddress?: Address;
+  annotation?: boolean;
+  createdAt?: number;
+}
+
+const ActionsPageFeedItem = ({
+  comment,
+  username,
+  walletAddress,
+  createdAt,
+  annotation = false,
+}: Props) => {
+  const { Decorate } = new TextDecorator({
+    username: (usernameWithAtSign) => (
+      <UserMention username={usernameWithAtSign.slice(1)} />
+    ),
+  });
+  const UserAvatar = HookedUserAvatar({ fetchUser: false });
+  return (
+    <div className={getMainClasses({}, styles, { annotation })}>
+      <div className={styles.avatar}>
+        <UserAvatar size="xs" address={walletAddress || ''} />
```

Comment:
The user avatar in the specs is `28x28` while yours is `26x26`

Reply:
But we don't have 28x28 avatars, till now we was using only options from Avatar appearance object to be consistent, do we want change it?

Reply:
I know we don,t but sometimes the Figma files have non-standard sizes. 

You can handle this in one of three ways:
- Talk to Karol about the discrepancy and see how this came about
- Just change the size locally in `ActionsPageFeedItem` and overwrite the `xs` size
- Leave it like it is as the size different is imperceptible

I would go with the latter as is the quickest. 

We'll most likely have some more time when wiring this, and we'll do a couple of polishing runs before launch